src/platform/packages/private/kbn-index-editor/src/index_update_service.ts

```diff
@@ -818,11 +820,20 @@ export class IndexUpdateService {
       await this.bulkUpdate(updates);
 
       this.setIndexCreated(true);
-      this.addAction('discard-unsaved-columns');
+      this.addAction('discard-unsaved-changes');
     } catch (error) {
       throw error;
     } finally {
       this._isSaving$.next(false);
     }
   }
+
+  public async exit() {
```

Comment:
the method is synchronous 
```suggestion
  public exit() {
```

Reply:
964ec116f00933922d8fd4309eb0fba2871a3b05